prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -334,6 +334,7 @@ enum fi_rdm_cm_role {
 
 struct fi_ibv_rdm_av_entry {
 	/* association of conn and EPs */
+	pthread_mutex_t			conn_lock;
```

Comment:
Is this structure per each entry in the AV, as the name suggests?  If so, how can it be used to protect the hash table?

Reply:
Yes, this structure (`struct fi_ibv_rdm_av_entry`) per each entry in the AV. But there is a hash in the AV entry that contains connection structures instances (`struct fi_ibv_rdm_conn`) per each Endpoint (i.e. key - EP, value - conn structure).
The `fi_ibv_domain::fi_ibv_rdm_cm::cm_lock` protects av_table and av_hash storages.

This solution is going to protect fi_ibv_rdm_av_entry::conn_hash storage.